Formula/bacula-fd@9.4.rb

```diff
@@ -0,0 +1,68 @@
+class BaculaFdAT94 < Formula
+  desc "Network backup solution"
+  homepage "https://www.bacula.org/"
+  url "https://downloads.sourceforge.net/project/bacula/bacula/9.4.4/bacula-9.4.4.tar.gz"
+  sha256 "0fe37a02ca768a720099d0d03509c364aff2390c05544d663f4819f8e7fc20be"
+
+
+  keg_only :versioned_formula
+
+  depends_on "openssl@1.1"
+  depends_on "readline"
+
+  uses_from_macos "zlib"
+
+  def install
+    # * sets --disable-conio in order to force the use of readline
+    #   (conio support not tested)
+    # * working directory in /var/lib/bacula, reasonable place that
+    #   matches Debian's location.
+    system "./configure", "--prefix=#{prefix}",
+                          "--sbindir=#{bin}",
+                          "--with-working-dir=#{var}/lib/bacula",
+                          "--with-pid-dir=#{var}/run",
+                          "--with-logdir=#{var}/log/bacula",
+                          "--enable-client-only",
+                          "--disable-conio",
+                          "--with-readline=#{Formula["readline"].opt_prefix}"
+
+    system "make"
+    system "make", "install"
+
+    # Avoid references to the Homebrew shims directory
+    inreplace Dir[prefix/"etc/bacula_config"],
+              HOMEBREW_SHIMS_PATH/"mac/super/", ""
+
+    (var/"lib/bacula").mkpath
+  end
+
+  def post_install
+    (var/"run").mkpath
+  end
+
+  plist_options :startup => true, :manual => "bacula-fd"
```

Comment:
This is not the same as the plist

Reply:
This PR is an almost straight copy of b[acula-fd](https://github.com/Homebrew/homebrew-core/blob/master/Formula/bacula-fd.rb) already in the repository, and the code in question is identical to what is already in the repository. I am not authorized to spend my time fixing the original committers omissions beyond what is required to get the software to work.